example/cmd/PasswordGenerator.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+
+	"github.com/bitfield/qrand"
+)
+
+func main() {
+
```

Comment:
Blank lines! You're breaking one of the BIT commandments, aren't you? (I don't know which one exactly. I think the first commandment would be "Thou shalt `gofmt` thy code", but blank lines are definitely in there somewhere.)